extension.js

```diff
@@ -168,6 +179,41 @@ const ExtensionLayout = new Lang.Class({
     GLib.spawn_command_line_async(cmd);
   },
 
+  _findNewStreamerEntries:function(lastList, currentList, detectGameChange) {
+    detectGameChange = detectGameChange || false;
+    if ( lastList.length == 0 )
+      return currentList;
+
+    let streamers = new Map();
+    lastList.forEach( ({streamer, game}) => streamers.set(streamer, game) );
+
+    return currentList.filter( ({streamer, game}) => {
+        if (!streamers.has(streamer))
+            return true;
+
+        if (detectGameChange &&
+            game != streamers.get(streamer))
+            return true;
+
+        return false;
+    });
+  },
+
+  _streamerOnlineNotification:function(streamer) {
+    let notification = new MessageTray.Notification(
+      this.notification_source,
+      _("%streamer% is live!").replace(/%streamer%/, streamer.streamer),
+      _("Playing %game%").replace(/%game%/, streamer.game));
```

Comment:
Yes, gettext should work like that. I haven't tried to localize it yet, but this should be fine.